src/scmdata/groupby.py

```diff
@@ -262,4 +261,57 @@ def reduce_array(ar):
             return self.apply(reduce_array)
 
 
+def get_joblib_parallel_processor(
+    n_jobs: int = -1,
+    backend: str = "loky",
+    *args: Any,
+    **kwargs: Any,
+) -> ParallelProcessor[RunLike, Q]:
+    """
+    Get parallel processor using :mod:`joblib` as the backend.
+
+    Parameters
+    ----------
+    n_jobs
+        Number of jobs to run in parallel (defaults to a single job which is useful for
```

Comment:
The default is all CPUs not a single worker.  Given that this requires more characters to set the number of jobs, I'm happy to default to all cores, but the docstring needs updated

Reply:
Updated now